### DISCUSSION TITLE
Mulltiple topics rule

### DISCUSSION THREAD

**Discussion by olabr** [emqx/emqx#14221] [General]
Hi.
I would like to create a rule that joins multiple topics into one new object topic. When I use multiple input topics only one topic is returned each "run". I would prefere not to subscribe to one topic containing the underlaying topics I need to access since this would cause a were large topic to subscribe to. How can I solve this?

**Comment by savonarola** [comment]
Hello! Sorry, could you clarify which kind of rule you refer to? Also, is it possible to give an example of the desired behavior?
